lib/ftu/rootpage.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import '../Navigation/app_controller.dart';
+import './authentication.dart';
+import './signuplogin.dart';
+
+enum AuthStatus {
+  NOT_DETERMINED,
```

Comment:
According to effective dart, [use lowerCamelCase for constant names](https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names).

```dart
enum AuthStatus {
  notDetermined,
  notLoggedIn,
  loggedIn,
}
```